tests/datasets/test_geo.py

```diff
@@ -33,11 +33,13 @@ def __init__(
         bounds: BoundingBox = BoundingBox(0, 1, 2, 3, 4, 5),
         crs: CRS = CRS.from_epsg(4087),
         res: float = 1,
+        paths: Optional[Union[str, Iterable[str]]] = None,
     ) -> None:
         super().__init__()
         self.index.insert(0, tuple(bounds))
         self._crs = crs
         self.res = res
+        self.paths = paths or []
```

Comment:
Does this work?
```suggestion
        paths: Union[str, Iterable[str]] = [],
    ) -> None:
        super().__init__()
        self.index.insert(0, tuple(bounds))
        self._crs = crs
        self.res = res
        self.paths = paths
```

Reply:
This would do the same, but I have heard that [mutable input defaults should be avoided](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/) :P